compiler/noirc_evaluator/src/brillig/mod.rs

```diff
@@ -65,6 +68,38 @@ impl Brillig {
             _ => unreachable!("ICE: Expected a function or procedure label"),
         }
     }
+
+    /// Converting an SSA function into Brillig bytecode.
+    pub(crate) fn convert_ssa_function(
```

Comment:
nit: this may make more sense inside of `brillig_fn.rs` like `convert_ssa_globals` is in `brillig_globals.rs` but this is non-blocking just noting it.

Reply:
The call stacks do span several brillig functions, so I think it's better here.